docs/cookbook/getFirstTuesdayOfMonth.mjs

```diff
@@ -6,7 +6,7 @@
  */
 function getFirstTuesday(queriedMonth) {
   // We first need to convert to a date
-  const firstOfMonth = queriedMonth.withDay(1);
+  const firstOfMonth = queriedMonth.toDate(1);
```

Comment:
Same here, maybe `toDateOn(1)`? (although that reads to me as less obvious than `toDateIn(2030)`)